node/Cargo.toml

```diff
@@ -21,26 +21,75 @@ authors = ["The Matter Labs Team <hello@matterlabs.dev>"]
 homepage = "https://matter-labs.io/"
 license = "MIT"
 
+[workspace.lints.rust]
+unsafe_code = "forbid"
+noop_method_call = "warn"
+missing_docs = "warn"
+unreachable_pub = "warn"
+unused_qualifications = "warn"
+unused_tuple_struct_fields = "warn"
+
+[workspace.lints.clippy]
+# restriction group
+create_dir = "warn"
+empty_structs_with_brackets = "warn"
+float_arithmetic = "warn"
+missing_docs_in_private_items = "warn"
+non_ascii_literal = "warn"
+partial_pub_fields = "warn"
+print_stdout = "warn"
```

Comment:
nit: can you move the lints stuff to the bottom of the Cargo.toml? It is not that important compared to dependencies specs.

Reply:
Sure.